install/amazon_rds/04_configure_the_collector_docker.mdx

```diff
@@ -44,21 +44,20 @@ Create your configuration file (e.g. named `pganalyze_collector.env`) with envir
 
 ## Handling Amazon Aurora clusters vs instances
 
-In the case of Amazon Aurora, the collector automatically resolves `cluster` endpoints to the underlying writer instance, and `cluster-ro` to the underlying reader instance:
+In the case of Amazon Aurora, the collector automatically resolves `cluster` endpoints to the underlying writer instance.
 
 ```
 DB_HOST=mydbcluster.cluster-123456789012.us-east-1.rds.amazonaws.com
 ```
 
-or
+This will only monitor the writer instance. If you also want to monitor a reader instance, you'll need to run another Docker container.
+Create a new configuration file (e.g. named `pganalyze_collector.replica.env`) and use the `cluster-ro` endpoint as the `DB_HOST`:
 
 ```
 DB_HOST=mydbcluster.cluster-ro-123456789012.us-east-1.rds.amazonaws.com
 ```
 
-For writer instances this is supported for any size of cluster, but for reader instances this is only supported in two-node clusters (i.e. single reader instance).
-
-If you have multiple readers you want to monitor you instead need to specify each instance endpoint separately, running one pganalyze collector Docker container for each instance:
+If you have multiple readers you want to monitor, you need to specify each instance endpoint separately, running one pganalyze collector Docker container for each instance:
```

Comment:
```suggestion
If you have multiple readers you want to monitor, you will need to specify each instance endpoint separately:
```
Though I guess we should update the example that follows to use the INI file syntax and have two separate sections. We can elide most of the config except the two `db_host` lines and the section names. We could use section names of `[primary]` and `[replica]` to make it clearer, too.